File: js/classificador.js
// Classificador de nível - JavaScript

const nomeDoHeroi = "Artemis"; // Troque pelo nome desejado
const xpDoHeroi = 8500;        // Troque pela XP desejada
let nivel = "";

if (xpDoHeroi < 1000) {
    nivel = "Ferro";
} else if (xpDoHeroi <= 2000) {
    nivel = "Bronze";
} else if (xpDoHeroi <= 5000) {
    nivel = "Prata";
} else if (xpDoHeroi <= 7000) {
    nivel = "Ouro";
} else if (xpDoHeroi <= 8000) {
    nivel = "Platina";
} else if (xpDoHeroi <= 9000) {
    nivel = "Ascendente";
} else if (xpDoHeroi <= 10000) {
    nivel = "Imortal";
} else {
    nivel = "Radiante";
}

console.log(`O Herói de nome ${nomeDoHeroi} está no nível de ${nivel}`);

File: python/classificador.py
# Classificador de nível - Python

nome_do_heroi = input("Digite o nome do herói: ")
xp_do_heroi = int(input("Digite a quantidade de XP: "))

if xp_do_heroi < 1000:
    nivel = "Ferro"
elif xp_do_heroi <= 2000:
    nivel = "Bronze"
elif xp_do_heroi <= 5000:
    nivel = "Prata"
elif xp_do_heroi <= 7000:
    nivel = "Ouro"
elif xp_do_heroi <= 8000:
    nivel = "Platina"
elif xp_do_heroi <= 9000:
    nivel = "Ascendente"
elif xp_do_heroi <= 10000:
    nivel = "Imortal"
else:
    nivel = "Radiante"

print(f"O Herói de nome {nome_do_heroi} está no nível de {nivel}")
